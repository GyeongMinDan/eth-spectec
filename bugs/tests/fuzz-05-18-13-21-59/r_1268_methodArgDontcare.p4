extern E<T> {
      E();
  }
   control c() {
      E<_>() e;
      apply {
     }
  }
