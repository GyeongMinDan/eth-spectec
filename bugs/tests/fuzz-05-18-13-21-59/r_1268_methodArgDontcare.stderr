likely_hits/r_1268_methodArgDontcare.p4(1): [--Wwarn=unused] warning: 'T' is unused
extern E<T> {
         ^
likely_hits/r_1268_methodArgDontcare.p4(4): [--Wwarn=unused] warning: Control c is not used; removing
   control c() {
           ^
[--Wwarn=missing] warning: Program does not contain a `main' module
